packages/client/bin/cli.ts

```diff
@@ -38,15 +40,13 @@ import type { BlockBytes } from '@ethereumjs/block'
 import type { GenesisState } from '@ethereumjs/blockchain/dist/genesisStates'
 import type { AbstractLevel } from 'abstract-level'
 
-const { hideBin } = require('yargs/helpers')
-const yargs = require('yargs/yargs')
-
 type Account = [address: Address, privateKey: Uint8Array]
 
 const networks = Object.entries(Common._getInitializedChains().names)
 
 let logger: Logger
 
+// @ts-ignore
```

Comment:
There is some type problem (which was just hidden before) occurring now that yargs has types: the `--dev` option is not so much liked by Yargs allowing for boolean and string values at the same time.

I nevertheless didn't dare to change anything here since I wasn't sure about the implications. Maybe @g11tech can chime in here, eventually also in a follow-up PR, since he has introduced most of these `--dev` switches for devnets I guess.

Reply:
will review although we have not been using `--dev` for a long time now